src/components/Dashboard/Content/index.tsx

```diff
@@ -1,14 +1,37 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 
+import { ReactComponent as ArrowRight } from 'assets/images/icons/arrow-right.svg';
 import Shimmer from 'components/Shimmer';
+import { Survey } from 'types/survey';
 
 interface DashboardContentProps {
+  surveys: Survey[];
+  currentPosition: number;
+  shouldShowShimmer: boolean;
+  onNextSurvey: () => void;
+  onIndicatorTapped: (position: number) => void;
   'data-test-id'?: string;
 }
 
-const DashboardContent = ({ ...attributes }: DashboardContentProps): JSX.Element => {
-  return (
-    <div className="flex flex-col w-full" {...attributes}>
+const DashboardContent = ({
+  surveys,
+  currentPosition,
+  shouldShowShimmer,
```

Comment:
```suggestion
  shouldShowShimmer = true,
```
we can make it `true` by default, no?